webui/react/src/components/ProjectCard.module.scss

```diff
@@ -2,18 +2,14 @@
   display: grid;
   gap: 16px;
   grid-template:
-    'header  header' auto
+    'header header' auto
     'workspace workspace' 20px
     'footer footer' 20px / auto auto;
   height: 100%;
 
-  .headerContainer {
+  h1 {
```

Comment:
prefer class over element selector?

Reply:
I can't apply a custom class to this section, but I will go with `:global(.ant-typography-single-line) {`